pkg/cmd/serverstatus/version.go

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
I would put this in `pkg/cmd/cli/version`.  I think it belongs under `cli` (I see that currently the version code is directly under `pkg/cmd`, but to me that doesn't make sense, as it's part of the CLI), and I would keep the package names aligned to command names.

Reply:
I agree with that too.